code/controllers/subsystem/mapping.dm

```diff
@@ -83,12 +89,17 @@ SUBSYSTEM_DEF(mapping)
 	setup_map_transitions()
 	generate_station_area_list()
 	QDEL_NULL(loader)
+	initialize_reserved_level()
 	..()
 
 /* Nuke threats, for making the blue tiles on the station go RED
    Used by the AI doomsday and the self destruct nuke.
 */
 
+/datum/controller/subsystem/mapping/fire()
+	if(clearing_reserved_areas)
```

Comment:
Something about this seems wrong, at a minimum you can use `can_fire` to avoid the overhead of this proc-call all the time, but also `clearing_reserved_areas` is never unset once it's set, and `wipe_turf_reservations()` has no `CHECK_TICK`s. It should probably be re-thought.

Reply:
oof.